client/web/src/enterprise/insights/pages/getting-started/components/code-insights-examples/CodeInsightsExamples.tsx

```diff
@@ -184,17 +189,17 @@ const CAPTURE_INSIGHT_EXAMPLES_DATA: LineChartContent<any, string> = {
 }
 
 const CAPTURE_GROUP_INSIGHT_CREATION_UI_URL_PARAMETERS = encodeCaptureInsightURL({
-    title: 'Terraform versions (present or most popular)',
+    title: 'Alpine versions over all repos',
     allRepos: true,
-    groupSearchQuery: 'app.terraform.io/(.*)\\n version =(.*)([0-9].[0-9].[0-9]) lang:Terraform archived:no fork:no',
+    groupSearchQuery: 'patterntype:regexp FROMs+alpine:([d.]+) file:Dockerfile',
```

Comment:
```suggestion
    groupSearchQuery: 'patterntype:regexp FROM\s+alpine:([\d\.]+) file:Dockerfile',
```
This query needs to be corrected